package.json

```diff
@@ -19,7 +19,7 @@
     "start": "node server/start.js",
     "start-watch": "nodemon server/start.js --watch server --watch db --watch index.js --watch package.json",
     "start-dev": "cross-env NODE_ENV=development npm run start-watch",
-    "test": "mocha --compilers js:babel-register --require babel-polyfill --watch-extensions js,jsx app/**/*.test.js app/**/*.test.jsx server/**/*.test.js db/**/*.test.js",
+    "test": "mocha --compilers js:babel-register --require babel-polyfill --watch-extensions js,jsx app/**/*.test.js app/**/*.test.jsx  app/components/**/*.test.jsx server/**/*.test.js db/**/*.test.js",
```

Comment:
I don't think you need this `app/components/**/*.text.jsx` line — you already have `/app/**/*/text.jsx`. The `**` glob includes recursive folders.